projects/views.py

```diff
@@ -484,6 +494,36 @@ def edit_nickname(request, project_id):
     return form
 
 
+
+@require_POST
+def edit_type(request, project_id):
+    form_args = {"request": request}
+    if not request.user.is_superuser:
```

Comment:
Very nice, I immediately started looking for this when i saw the `{% is_admin %}` in the template :) -- good that we are checking on both the client and the server.